backend/msgpack/server.go

```diff
@@ -182,12 +194,45 @@ func (s *server) write(
 		s.RUnlock()
 		return err
 	}
-	numShards := placement.NumShards()
-	shard := s.shardFn(id, numShards)
-	instances := placement.InstancesForShard(shard)
-	err = s.writerMgr.WriteTo(instances, shard, mu, pl)
+	var (
+		numShards = placement.NumShards()
+		shardID   = s.shardFn(id, numShards)
+		instances = placement.InstancesForShard(shardID)
```

Comment:
I see, so then the call to `ActivePlacement` in line 191 above will switch placements after the cutover time?